src/composables/transactionTx.ts

```diff
@@ -38,14 +39,14 @@ export function useTransactionTx({
 
   const outerTx = ref<ITx | undefined>(tx);
   const innerTx = ref<ITx | undefined>(tx ? getInnerTransaction(tx) : undefined);
-  const ownerAddress = ref<string | undefined>(externalAddress);
+  const ownerAddress = ref<Encoded.AccountAddress | undefined>(externalAddress);
 
   function setTransactionTx(newTx: ITx) {
     outerTx.value = newTx;
     innerTx.value = getInnerTransaction(newTx);
   }
 
-  function setExternalAddress(address: string) {
+  function setExternalAddress(address: Encoded.AccountAddress) {
     ownerAddress.value = address;
   }
```

Comment:
Function name is confusing. Name is `setExternalAddress` but it is setting `ownerAddress`

Reply:
I don't think it should be tackled in this PR.